.gitignore

```diff
@@ -1,5 +1,5 @@
 ## Ignore WebConfig because it contains sensitive values
-Web.config
+Authy2FA/Web.config
```

Comment:
Adding web.config to git ignore means the project does not run out of the box from Github.  The project thinks it has a file called web.config but it actually doesn't.